### ISSUE TITLE
Consider switching timing to time.monotonic()

### ISSUE THREAD

**Issue by icook** [simplecrypto/powerpool#84]
All timing currently relies on system clock. It would be better to use monotonic.

https://docs.python.org/3/library/time.html#time.monotonic

**Comment by ericecook** [comment]
This requires Python 3 unless you want to use something like https://pypi.python.org/pypi/monotonic/

**Comment by icook** [comment]
Yeah, it's mostly a pipe dream since I don't remember feeling good about the quality of that module. Ideally we would switch to Python3 with `guv`, but that would require a big rewrite in some places. There's not a great resolution for this, but it's consequences are pretty minor as long as ntpd is being run.

**Comment by ericecook** [comment]
Yea I hear that. Last time I looked at `guv` it was still a bit greener than I'd like, although it looks like dev work is ongoing
